fixtures/get-bindings-proxy/tests/get-bindings-proxy.test.ts

```diff
@@ -227,6 +228,23 @@ describe("getBindingsProxy", () => {
 			await dispose();
 		}
 	});
+
+	describe.only("caches", () => {
+		it("correctly obtains functioning caches", async () => {
+			const { caches, dispose } = await getBindingsProxy<Bindings>({
+				configPath: wranglerTomlFilePath,
+			});
+			try {
+				let cache = caches.default;
+				const noMatch = await cache.match("http://0.0.0.0/test");
+				expect(noMatch).toBeUndefined();
+
+				await cache.put("http://0.0.0.0/test", new Response("test"));
```

Comment:
@mrbbot This put generates the following error:
![Screenshot 2024-01-27 at 00 20 05](https://github.com/cloudflare/workers-sdk/assets/61631103/f966ea34-1a85-4949-b2a3-c20b0d1ac393)

Am I doing something wrong? Could you advice me on how to use the caches from miniflare? 🙏